### DISCUSSION TITLE
@nuxtbot nuxtbot changed the title Is there a way for Nuxt to render every page in SSR mode, rather than just the first time the page is opened?

### DISCUSSION THREAD

**Discussion by robertyclin** [nuxt/nuxt#27758] [Questions]
**Summary (generated)**:

Here is a summary of the text:

Is there a way for Nuxt to render every page in Server-Side Rendering (SSR) mode, similar to PHP. The speaker is open to helping implement this feature and thinks it could be implemented as a module. Before proceeding, the speaker has checked the contribution guide and existing discussions and issues on GitHub.

<hr>

### Describe the feature

Is there a way for Nuxt to render every page in SSR mode，like php。

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by robertyclin** [comment]
Could you provide an example?

**Comment by TheAlexLichter** [comment]
I wonder why? 

But yes, you can:

1. Use `a` links (or `<NuxtLink external to="...">`) everywhere to do so (still have JS but no SPA routing)
2. Or remove all js via https://nuxt.com/docs/guide/going-further/features#noscripts
